napkon_string_matching/terminology/loinc_search.py

```diff
@@ -12,9 +14,56 @@
 RESPONSE_NO_ENTRIES = "Keine passenden Einträge gefunden"
 RESPONSE_LOGIN = "Log In ‹ LOINC — WordPress"
 
+STOP_WORDS = set(stopwords.words('german'))
+
+EXAMPLEOUTCOME_ITEMS = [
+    """Dialyse:""",
+    """Invasiver Beatmung:""",
+    """Sauerstoff:""",
+    """Trachealkanüle:""",
+    """Respiratorisches Outcome (Sauerstoff-Therapie):"""]
```

Comment:
I do not see the use-case of triple quotes here. Is there any reason? Otherwise I would like to use "normal" quotes.